tag KJT related part to sc.INPUT_DIST

jiyuanzFB opened the pull request:
Summary: tag KJT output to sc.INPUT_DIST to enforce placement on CPU

Differential Revision: D67313214

jingsh commented on torchrec/quant/embedding_modules.py:
```diff
@@ -300,6 +300,23 @@ def _fx_trec_unwrap_kjt(
         return indices.int(), offsets.int()
 
 
+@torch.fx.wrap
```
this line not needed

jingsh commented on torchrec/quant/embedding_modules.py:
```diff
@@ -300,6 +300,23 @@ def _fx_trec_unwrap_kjt(
+) -> Tuple[torch.Tensor, torch.Tensor]:
+    """
+    Forced conversions to support TBE
+    CPU - int32 or int64, offsets dtype must match
+    GPU - int32 only, offsets dtype must match
```
can you provide reference on this dtype?

jingsh reviewed (approved):
nits

jyingl3 committed:
tag KJT related part to sc.INPUT_DIST (#2678)

Summary:
Pull Request resolved: https://github.com/pytorch/torchrec/pull/2678

tag KJT output to sc.INPUT_DIST to enforce placement on CPU

Differential Revision: D67313214
